ui/src/components/ModelBar.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Row } from 'components/Layout';
+
+export default ({ name }) => (
+  <div>
+    <div
+      css={`
+        height: 6px;
+        box-shadow: 0 0 4.9px 0.1px #bbbbbb;
+        background-color: #ffffff;
+        border: solid 1px #e0e1e6;
+      `}
+    />
+
+    <Row className="model-bar">
+      <h2>Model {name}</h2>
+      <Row
```

Comment:
Being able to write

```
<Row alignItems="center">
```

is kind of nice

Reply:
here are all the options: https://github.com/jxnblk/styled-system#table-of-style-props